StreamVideoTests/Utilities/Dummy/CallResponse+Dummy.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+
+extension CallResponse {
```

Comment:
I think we already had a way to create some of these responses.

Reply:
Really? Can you elaborate?

Reply:
check the `MockResponseBuilder`